src/core/prompt_builder.py

```
@@ -231,6 +231,36 @@ def load_template(template_name: str = "somas_prompt.txt") -> str:
         return f.read()
 
 
+# --- Faktencheck-Verifikation (v0.10.0) ---
+
+# Single Source of Truth für das parsbare Stufe-1-Format. Enthält die exakten
+# Vertrags-Marker (**Meinungen:** / **Interpretationen:** / **Behauptungen
+# (überprüfbar):**), auf die der Claim-Parser (extract_claims_from_faktencheck)
+# angewiesen ist. Wird bei erzwungenem FAKTENCHECK injiziert (s. _apply_custom_overrides),
+# damit das Format in JEDEM Preset garantiert ist – auch in den namens-only-Templates.
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)